.github/README.md

```diff
@@ -0,0 +1,97 @@
+# Intel managed distribution of Istio
+
+## Introduction
+
+Intel managed distribution of Istio is a project aiming to showcase and
+integrate various Intel technologies into Istio and Envoy. The focus is
+in letting both upstream community and users know what Intel is working
+on, finding gaps in upstream project features in relation to hardware
+enablement, and testing and deploying Intel features for Istio service
+mesh.
+
+Intel managed distribution of Istio consists of the following source
+code repositories:
+
+* https://github.com/intel/envoy
+
+* https://github.com/intel/envoy-go-control-plane
+
+* https://github.com/intel/istio
+
+* https://github.com/intel/istio-api
+
+* https://github.com/intel/istio-proxy
+
+## Project goals and relation to upstream projects
+
+The goal of the project is not to maintain permanent Istio and Envoy
+forks, but rather have a place to test and maintain features which can
+be later upstreamed. When features are added to Istio and Envoy
+upstream, they will be removed from this distribution, reducing the
+difference to upstream. We intend to have Intel distribution for Istio
+always compatible with the usual APIs, configuration formats, and
+tooling. We will just extend Istio and Envoy with new extensions and
+APIs.
+
+## Release branch policy
+
+Intel distribution for Istio will track the latest Istio release branch
+in the corresponding branch. For example, Istio 1.15 will be tracked in
+release-1.15-intel branch. The releases will be tagged with a naming
```

Comment:
We might want to update the example branches, but since they are just examples, it's not critical I think.